Vic Piña/HW01/factoriales.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+using namespace std;
+
+int main() {
```

Comment:
Código es in correcto, el segundo for nunca entra. Al multiplicar por 0, siempre obtendrás 0.